test/e2e/metrics_util.go

```diff
@@ -257,15 +259,19 @@ func getMetrics(c *client.Client) (string, error) {
 }
 
 // Retrieves scheduler metrics information.
-func getSchedulingLatency() (SchedulingLatency, error) {
+func getSchedulingLatency(c *client.Client) (SchedulingLatency, error) {
 	result := SchedulingLatency{}
 
-	cmd := "curl http://localhost:10251/metrics"
-	sshResult, err := SSH(cmd, getMasterHost()+":22", testContext.Provider)
-	if err != nil || sshResult.Code != 0 {
-		return result, fmt.Errorf("unexpected error (code: %d) in ssh connection to master: %#v", sshResult.Code, err)
-	}
-	samples, err := extractMetricSamples(sshResult.Stdout)
+	data, err := c.Get().
+		Prefix("proxy").
+		Namespace(api.NamespaceSystem).
+		Resource("pods").
+		Name(fmt.Sprintf("kube-scheduler-%v:%v", testContext.CloudConfig.MasterName, ports.SchedulerPort)).
```

Comment:
I expect this to break in GKE (scheduler mirror pod won't be registered with master). Is this run in the GKE tests?

@roberthbailey